tests/functional/oauth_sign_up.js

```diff
@@ -88,6 +90,58 @@ registerSuite('oauth signup', {
         .then(noEmailExpected(email, 1));
     },
 
+    'signup in Chrome for Android, verify same browser': function () {
```

Comment:
I tried adding the same tests in the signin flow, but forget how to force a verification loop.

Reply:
`but forget how to force a verification loop.`

 "SIGNIN_CONFIRMATION_SKIP_FOR_NEW_ACCOUNTS": "false" in auth-server config